compiler-cli/src/main.rs

```diff
@@ -188,6 +188,7 @@ enum Command {
         #[arg(short, long, ignore_case = true)]
         target: Option<Target>,
 
+        /// Optionally specify the JavaScript runtime: nodejs, deno, bun
```

Comment:
This list of variants needs to be generated from the type itself rather than being a comment.

Reply:
I've been looking and I don't think that's possible due to it being an `Option<Runtime>` as opposed to just `Runtime`. I'll keep looking though.

Reply:
I solved it by using `#[arg(help = runtime_doc()]` where `runtime_doc` is a new function I made in `compiler-cli/src/main.rs`. This could be changed to a method that's attached to the `Runtime` struct (maybe `Runtime::help_doc()`?)